Misc/NEWS.d/next/Library/2018-09-23-03-18-52.bpo-34775.vHeuHk.rst

```diff
@@ -0,0 +1,2 @@
+Division handling of PurePath now returns NotImplemented instead of raising
+a TypeError when passed something other than an instance of str or PurePath.
```

Comment:
Please add "Path by *yourname*."

Reply:
I assumed you meant "Patch by" from the other items in the change log.